recipes/ois/all/patches/001_fix.patch

```diff
@@ -0,0 +1,25 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -213,7 +213,7 @@
+ set(LIB_INSTALL_DIR "lib")
+ set(OIS_LIB_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR})
+
+-file(GLOB PUBLIC_HEADERS "${CMAKE_SOURCE_DIR}/includes/*.h")
++file(GLOB PUBLIC_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/includes/*.h")
+
+ install(FILES ${PUBLIC_HEADERS}
+   DESTINATION include/ois
+@@ -225,12 +225,3 @@
+   ARCHIVE DESTINATION lib
+   FRAMEWORK DESTINATION Library/Frameworks
+ )
+-
+-if(MSVC)
+-  install(FILES $<TARGET_PDB_FILE:OIS> DESTINATION bin OPTIONAL)
+-endif(MSVC)
+-
+-if(UNIX)
+-    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/OIS.pc.in ${CMAKE_BINARY_DIR}/OIS.pc @ONLY)
+-    install(FILES ${CMAKE_BINARY_DIR}/OIS.pc DESTINATION ${OIS_LIB_DIRECTORY}/pkgconfig)
+-endif()
+\ No newline at end of file
```

Comment:
For easier maintenance of this recipe,
I think it is easier to just remove the installed files/dirs in `package` instead of patching the cmakescript.